server/src/main/java/com/objectcomputing/checkins/services/member_skill/skillsreport/SkillsReportServices.java

```diff
@@ -0,0 +1,5 @@
+package com.objectcomputing.checkins.services.member_skill.skillsreport;
+
+public interface SkillsReportServices {
+    SkillsReportResponseDTO report(SkillsReportRequestDTO request);
```

Comment:
I don't think I like using the DTO directly here. DTO is really a transport thing and this is the service API. I think I would pass in the individual parameters needed by the method. If we are doing this elsewhere in the app, we should likely create stories to clean that up.

I could be convinced otherwise on this...it's a grey area. Especially given that I would be inclined to share the response object in this case. It can always be refactored. 🤷 (bringing comment over from previous PR)

Reply:
Yeah, I saw other services not using DTO directly as parameters but wasn't sure what would be a good practice. I'll change the input parameters. Is returning `SkillsReportResponseDTO` ok or should I change its type?

Reply:
I think I would make drop the DTO from it's name (the response obj), since it's being returned from the service. if at some point we need to return something organized differently from the controller we will call that thing a DTO